tests/components/group/test_notify.py

```diff
@@ -166,6 +167,34 @@ async def test_send_message_with_data(hass: HomeAssistant, tmp_path: Path) -> No
     )
     send_message_mock.reset_mock()
 
+    # Test sending a message to a notify group which does not have title.
+    await hass.services.async_call(
+        "notify",
```

Comment:
`notify.my_notification_group` is the legacy service not used by notify entities. Use the `notify.send_message` service.